reconcile/quay_org.py

```diff
@@ -0,0 +1,184 @@
+import logging
+
+import reconcile.gql as gql
+import utils.vault_client as vault_client
+from utils.quay_api import QuayApi
+
+from reconcile.aggregated_list import (AggregatedList,
+                                       AggregatedDiffRunner,
+                                       RunnerException)
+
+QUAY_ORG_CATALOG_QUERY = """
```

Comment:
The app-interface schemas contain the list of managed quay orgs and teams as specified in this schema:
https://github.com/app-sre/qontract-server/pull/11/files#diff-a3f753bf6d00d416297a369b20e319e7